### ISSUE TITLE
Server list reporting bug

### ISSUE THREAD

**Issue by okanozdemir** [TeaSpeak/TeaSpeak#576]
**Describe the bug**
Even if you have turned off reporting to server list from `Edit Virtual Server -> Misc -> Enable reporting to server list` or YaTQA the server is still on the list.

**Expected behaviour**
The teaspeak server should not appear in the list.

**Screenshots**
![Edit Virtual Server -> Misc -> Enable reporting to server list](https://user-images.githubusercontent.com/6042324/53689017-8ab6cf00-3d5d-11e9-8241-0e77654a85e1.png)

**Environment information:**
 Operating system: Linux #34-Ubuntu SMP x86_64 GNU/Linux
 TeaSpeak version: 1.3.9-beta
 I am using `experimental_31`: 1

**Comment by WolverinDEV** [comment]
You've to wait 10 min 'till the server expires :)

**Comment by okanozdemir** [comment]
Yes it did not appear as you said but I'm using this feature off for a long time. I didn't understand how it activated.
